### ISSUE TITLE
ubuntu 16.04 and keyring issue

### ISSUE THREAD

**Issue by ghamed** [ceph/ceph-ansible#1320]
Hi,

I have the following environment:

OS 16.04 (Latest)

3 * OSD
3 * MON
1 * admin/mgmt
1 * client

Public Network : 192.168.0.0/24
Cluster Network: 10.0.0.0/24

however everytime i run the play book  `ansible-playbook -K -b site.yml` i got the following

```
TASK [ceph-mon : test if initial mon keyring is in mon kv store] ***************
fatal: [mon1]: FAILED! => {"changed": false, "cmd": ["ceph", "--cluster", "ceph", "config-key", "get", "initial_mon_keyring"], "delta": "0:05:00.083971", "end": "2017-02-20 15:47:46.290105", "failed": true, "rc": 1, "start": "2017-02-20 15:42:46.206134", "stderr": "2017-02-20 15:47:46.253960 7fc6aebd1700  0 monclient(hunting): authenticate timed out after 300\n2017-02-20 15:47:46.253983 7fc6aebd1700  0 librados: client.admin authentication error (110) Connection timed out\nError connecting to cluster: TimedOut", "stdout": "", "stdout_lines": [], "warnings": []}
```

Also when i ssh to the mon nodes and when i run ceph -s it times out.

Also following is the config files.

[configfiles.zip](https://github.com/ceph/ceph-ansible/files/788131/configfiles.zip)

**Comment by leseb** [comment]
Is the monitor started?

**Comment by ghamed** [comment]
Hi,

monitors are started. i even tried the commands manually but still timing out.

**Comment by leseb** [comment]
What's your ansible version?

**Comment by ghamed** [comment]
ansible 2.0.0.2

**Comment by leseb** [comment]
Please try with 2.2.1.0

**Comment by ommoreno** [comment]
I also see this issue with ansible 2.2.1.0 and ceph-ansible >=2.1.0

**Comment by ghamed** [comment]
The issue i originally had has been resolved by updating the ansible to latest 2.2.

Thanks again for the help.
